FDK/Tools/SharedData/FDKScripts/CheckOutlinesUFO.py

```diff
@@ -12,1020 +13,1145 @@
-for a curve define a flat curve, and whether a sequence of line segments
-defines a straight line.
+-setTolerance <n>
+    Set the maximum deviation from a straight line allowed.
+    Default is 0 design space units. This is used to test whether the control
+    points for a curve define a flat curve, and whether a sequence of line
+    segments defines a straight line.
 
--wd	write changed glyphs to default layer instead of '%s'
+-wd
+    write changed glyphs to default layer instead of '%s'
 
 -g <glyphID1>,<glyphID2>,...,<glyphIDn>
-	Check only the specified list of glyphs. The list must be
-	comma-delimited. The glyph ID's may be glyphID's, glyph names, or
-	glyph CID's. If the latter, the CID value must be prefixed with the
-	string "cid". There must be no white-space in the glyph list.
-	Examples:
-		checkOutlinesUFO -g A,B,C,69 myFont
-		checkOutlinesUFO -g cid1030,cid34,cid3455,69 myCIDFont
+    Check only the specified list of glyphs. The list must be
+    comma-delimited. The glyph ID's may be glyphID's, glyph names, or
+    glyph CID's. If the latter, the CID value must be prefixed with the
+    string "cid". There must be no white-space in the glyph list.
+    Examples:
+        checkOutlinesUFO -g A,B,C,69 myFont
+        checkOutlinesUFO -g cid1030,cid34,cid3455,69 myCIDFont
 
 -gf <file name>
-	Check only the list of glyphs contained in the specified file, The file
-	must contain a comma-delimited list of glyph identifiers. Any number of
-	space, tab, and new-line characters are permitted between glyph names
-	and commas.
+    Check only the list of glyphs contained in the specified file, The file
+    must contain a comma-delimited list of glyph identifiers. Any number of
+    space, tab, and new-line characters are permitted between glyph names
+    and commas.
 
--all Force all glyphs to be processed. This applies only to UFO fonts, where processed glyphs
-	are saved in a layer, and processing of a glyph is skipped if has already been processed.
+-all
+    Force all glyphs to be processed. This applies only to UFO fonts, where
+    processed glyphs are saved in a layer, and processing of a glyph is skipped
+    if has already been processed.
 
 """ % kProcessedGlyphsLayerName
 
 __doc__ = __usage__ + __help__
 
-
 import sys
 import os
 import re
-import copy
 import defcon
 import ufoLib
 import subprocess
 import shutil
 import ufoTools
 import booleanOperations.booleanGlyph
 from robofab.pens.digestPen import DigestPointPen
-from plistlib import readPlist, writePlistToString
 
 import hashlib
 
+nameAliasDict = {}
+
 kSrcGLIFHashMap = "com.adobe.type.checkOutlinesHashMap"
 
-class focusOptionParseError(KeyError):
-	pass
 
-class focusFontError(KeyError):
-	pass
+class FocusOptionParseError(KeyError):
+    pass
+
+
+class FocusFontError(KeyError):
+    pass
+
 
 kUknownFontType = 0
 kUFOFontType = 1
 kType1FontType = 2
 kCFFFontType = 3
 kOpenTypeCFFFontType = 4
```

Comment:
please change all these names to uppercase with underscore separators

Reply:
OK. Am I correct in thinking I can drop the initial "k"?

Reply:
absolutely